app/Actions/Album/SetProtectionPolicy.php

```diff
@@ -29,12 +29,18 @@ class SetProtectionPolicy extends Action
 	 */
 	public function do(BaseAlbum $album, AlbumProtectionPolicy $protectionPolicy, bool $shallSetPassword, ?string $password): void
 	{
-		$album->grants_full_photo = $protectionPolicy->grantsFullPhoto;
-		$album->is_public = $protectionPolicy->isPublic;
-		$album->requires_link = $protectionPolicy->requiresLink;
-		$album->is_nsfw = $protectionPolicy->isNSFW;
-		$album->is_downloadable = $protectionPolicy->isDownloadable;
-		$album->is_share_button_visible = $protectionPolicy->isShareButtonVisible;
+		// Security attributes of the album itself independent of a particular user
+		// Note: The first one (`is_public`) will become implicit in the future when the following three attributes are
+		// move to a separate table for sharing albums with anonymous users
+		$album->is_public = $protectionPolicy->is_public;
+		$album->is_link_required = $protectionPolicy->is_link_required;
+		$album->is_nsfw = $protectionPolicy->is_nsfw;
+
+		// (Future) permissions on an album-user relation.
+		// Note: For the time being these are still "globally" defined on the album for all users, but they will be
+		// moved to a separate table for sharing albums with users.
+		$album->grants_download = $protectionPolicy->grants_download;
+		$album->grants_access_full_photo = $protectionPolicy->grants_access_full_photo;
```

Comment:
```suggestion
		$album->grants_full_photo_access = $protectionPolicy->grants_full_photo_access;
```

A zombie. :-( This one had already been resolved on the predecessor PR and now it is back.

Reply:
Fixed [31a5e2b](https://github.com/LycheeOrg/Lychee/pull/1539/commits/31a5e2b97c5302595e7b13326c54d6980c16498b)